backend/go.mod

```diff
@@ -28,6 +28,8 @@ replace github.com/miekg/dns => github.com/miekg/dns v1.1.25
 
 replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.7
 
+replace github.com/highlight/highlight/sdk/highlight-go => ../sdk/highlight-go
```

Comment:
@Vadman97 I think this fixed the issue we were running into. Props to @mayberryzane for finding this!

Reply:
amazing, and TIL!